### PR TITLE
Updates the sprites for bullets (inflight), adds pellet sprites, redoes the thermal projectile sprites

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Bullet improvements. Yeah.

[vid of the bullets inflight](https://streamable.com/2cxv86)

[Updated video of the bullets and pellets](https://streamable.com/dim0xc)

[Original bullets for comparison](https://streamable.com/2j3yc7)

## Why It's Good For The Game

Replacing this with an actual explanation, having looked at it more closely, our current ballistic bullets which represent a good deal of our projectiles is literally just a fucking tiny black dot.

Some of these bullets range from a measly 10 damage to as high as 60 or 70. 

I think you should be able to see a bullet even briefly before impact. It's not realistic but so is being able to see a bullet in flight at all.

## Changelog

:cl:
imageadd: Improves the sprites for bullets, thermal projectiles, and introduces pellet sprites.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
How's it look in action

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 
 /obj/projectile/bullet/shotgun_slug/executioner
 	name = "executioner slug" // admin only, can dismember limbs
+	icon_state = "pellet"
 	sharpness = SHARP_EDGED
 	wound_bonus = 80
 
 /obj/projectile/bullet/shotgun_slug/pulverizer
 	name = "pulverizer slug" // admin only, can crush bones
+	icon_state = "pellet"
```
Comment (Fikou) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:16]: these are inherited from parent

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 2]
```diff
@@ -24,15 +28,18 @@
 
 /obj/projectile/bullet/incendiary/shotgun
 	name = "incendiary slug"
+	icon_state = "pellet"
 	damage = 20
 
 /obj/projectile/bullet/incendiary/shotgun/no_trail
 	name = "precision incendiary slug"
+	icon_state = "pellet"
```
Comment (Fikou) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:36]: and these

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 3]
```diff
@@ -24,15 +28,18 @@
...
 	damage = 35
 	leaves_fire_trail = FALSE
 
 /obj/projectile/bullet/incendiary/shotgun/dragonsbreath
 	name = "dragonsbreath pellet"
+	icon_state = "pellet"
```
Comment (Fikou) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:42]: and these

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 4]
```diff
@@ -1,21 +1,25 @@
 /obj/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
+	icon_state = "pellet"
 	damage = 50
 	sharpness = SHARP_POINTY
 	wound_bonus = 0
 
 /obj/projectile/bullet/shotgun_slug/executioner
 	name = "executioner slug" // admin only, can dismember limbs
+	icon_state = "pellet"
```
Comment (Fikou) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:10]: and these

File: code/modules/projectiles/projectile/bullets/shotgun.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/projectiles/projectile/bullets/shotgun.dm:16]: bah

**Comment by necromanceranne** [comment]
> How's it look in action

Almost a meaningless distinction because of how fast most projectiles move.

**Comment by necromanceranne** [comment]
What I was being honest :P

I'll get a vid up in a bit I'm just being a shit.

**Comment by Wallemations** [comment]
> > How's it look in action
> 
> 
> 
> Almost a meaningless distinction because of how fast most projectiles move.

If that was the case what's the point of your PR lol

**Comment by necromanceranne** [comment]
> > > How's it look in action
> > 
> > 
> > Almost a meaningless distinction because of how fast most projectiles move.
> 
> If that was the case what's the point of your PR lol

For those odd times I spy the pixel thin bullet inflight and die a little.

**Comment by necromanceranne** [comment]
> How's it look in action

[Here you go.](https://streamable.com/2cxv86)

**Comment by timothymtorres** [comment]
I'm going to be honest, the sprites zoomed in look awesome but when they are zoomed out in normal view for the video they look chunky and too big.

-1 from me

**Comment by necromanceranne** [comment]
Could actually be more a colour problem. I'll darken the palette and see how that goes.

The thermal bullets currently are actually bigger than the ones in this pr, fyi.

**Comment by VladinXXV** [comment]
I dig the changes for thermal pistol bullets, but I find regular bullets too big as well. I suppose we'll see what a darker palette can do for it, but there is a pretty large size difference just looking at the sprites out of game. I think you can still improve the appearance without such a massive size difference. It's not like the competition is beautiful by any standards.

**Comment by necromanceranne** [comment]
True, a pixels width difference probably won't change much

**Comment by MrDoomBringer** [comment]
Smaller bullets are way better, even if darkened i suspect these will look like golf balls in comparison

**Comment by necromanceranne** [comment]
> Smaller bullets are way better, even if darkened i suspect these will look like golf balls in comparison

Anything is going to look larger than what is effectively a moving thin black dot, that's straight up a given. What you normally actually probably see first with the current sprite is the red streak behind the bullet.

**Comment by necromanceranne** [comment]
anyway I shrunk the sprite and made it darker, you can see the new changes in the videos I posted and also comparison to the old sprite

**Comment by realthomas69** [comment]
they're too chunky

**Review by Twaticus — APPROVED** [review]

**Comment by ShizCalev** [comment]
>making animated sprites not animated anymore

:(

**Comment by necromanceranne** [comment]
> > making animated sprites not animated anymore
> 
> :(

It's a two frame animation. It's barely an animation.

**Comment by MrDoomBringer** [comment]
In most video games, not to mention real life, bullets are small and slightly hard to see, but I never really had a problem with seeing them in ss13. Is there any meaningful data driving this change other than it's hard for you to see the bullets so you made them larger?

**Comment by necromanceranne** [comment]
This is a video game.

**Pull request merged**